library/library_example.cc

```diff
@@ -12,7 +13,8 @@ inline VW::feature vw_feature_from_string(VW::workspace& v, const std::string& f
 
 int main(int argc, char* argv[])
 {
-  VW::workspace* model = VW::initialize("--hash all -q st --noconstant -f train2.vw --no_stdin");
+  auto model = VW::initialize(VW::make_unique<VW::config::options_cli>(
```

Comment:
Nit: Splitting into two lines seems more reading friendly to me.

Reply:
It's formatted with clang-format, I think it's better to leave it to the tool